### PR TITLE
GSoC: Week 12 Blogpost

### PR DESCRIPTION
**Pull request by JoaoDell**
Hello everyone, here is my week 12 blogpost!
image

### CODE REVIEW

File: docs/source/posts/2023/2023-08-21-week-12-joaodellagli.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Week 12: Now That is (almost) a Wrap!
+==========================
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-08-21-week-12-joaodellagli.rst:2]: `=` symbol should cover the whole title

File: docs/source/posts/2023/2023-08-21-week-12-joaodellagli.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+So how did it go?
+-----------------
+All went fine with no difficult at all, thankfully.
+
+The Next Steps
+--------------
+My next plans are, after having PR `#826 <https://github.com/fury-gl/fury/pull/826>`_ merged, to work on the float encoding issue described in 
+:doc:`this blogpost<2023-07-31-week-9-joaodellagli>`. Also, I plan to tackle the UI idea once again, to see if I can finally give the user 
+a way to control de intensities of the distributions.
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-08-21-week-12-joaodellagli.rst:37]: `the` instead of `de`

**Review by ganimtron-10 — CHANGES_REQUESTED** [review]
Hey @JoaoDell ,
I had these below comments. PTAL.

**Review by skoudoro — APPROVED** [review]
All good, merging

**Pull request merged**